src/scss/objects/_buttons.scss

```diff
@@ -251,45 +279,19 @@ $btn-rounded-width: 38px !default;
     &:active,
     &.is-active {
         outline: none; // no need as already has a focus state
-        background-color: $btn-outline-bgColor--hover;
-        color: $btn-outline-textColor--hover;
+        background-color: $btn-outline-bgColor--highlight;
+        color: $btn-outline-textColor--highlight;
     }
 
-    &:focus,
-    &:hover {
+    &:hover,
+    &:focus {
         border: 1px solid $btn-outline-borderColor--hover;
     }
     &:active {
         border: 1px solid $btn-outline-borderColor--active;
     }
 }
 
-/**
- * Transparent Button
```

Comment:
Where did these styles go?

Reply:
And a load below. Looks they have been removed?

Reply:
Yep – they've been removed because they aren't in the global buttons defined by the design team.  They got ported from international but shouldn't have done.

If we need them back in as part of work on new pages that gets done we can add them back in as they get created by design.